cypress/integration/adminFlows/config/authenticationSection.spec.js

```diff
@@ -20,7 +20,7 @@ describe('Authentication Section', () => {
           facebookKey: 'somekey',
           facebookSecret: 'somesecret',
         }).then(() => {
-          cy.visit('/admin/config');
```

Comment:
update cypress tests to point to new admin routes. We can't use path helpers here unfortunately :(

Reply:
I wonder if we should make our own so that you can call something like `railsRoutes.admin_config`. Creating this JS utility class could probably at least be semi-automated and if any routes ever change again we only have to update them in a single place. cc @nickytonline